runtime/queries/javascript/textobjects.scm

```diff
@@ -0,0 +1,36 @@
+(function_declaration
+  body: (_)) @function.around
+
+(function
+  body: (_)) @function.around
+
+(arrow_function
+  body: (_)) @function.around
+
+(method_definition
+  body: (_)) @function.around
```

Comment:
for all of these we can mark the body node as `function.inside`, for example:

```
(function
   body: (_) @function.inside) @function.around
```

Reply:
I don't quite understand mark `function.inside` .What is the difference between inside and not marking it?

Reply:
You can use `mif` (inside) to select inside the function. For most (maybe all?) languages this selects the body of the function while `maf` (around) also selects things like the function name and any function keywords

Reply:
thanks, i just test like `maf` and `]f` Navigating...